playbooks/common/openshift-master/config.yml

```diff
@@ -275,11 +267,29 @@
                                       | map(attribute='stdout')
                                       | list) }}"
 
+- name: Configure master named certificates
+  hosts: oo_masters_to_config
+  vars:
+    named_certs_specified: "{{ openshift_master_named_certificates is defined }}"
+  tasks:
+  - name: Land named certificates
+    copy: src="{{ item.certfile }}" dest="{{ master_cert_config_dir }}"
+    with_items: openshift_master_named_certificates
+    when: named_certs_specified | bool
+  - name: Land named certificate keys
+    copy: src="{{ item.keyfile }}" dest="{{ master_cert_config_dir }}"
+    with_items: openshift_master_named_certificates
+    when: named_certs_specified | bool
```

Comment:
Alright so, we don't land the files if the source is missing and we fail if the destination is missing after the copies (that may or may not have happened)? Does that sound right?